ci/conda/recipes/morpheus/morpheus_build.sh

```diff
@@ -32,6 +32,15 @@ if [[ -n "${MORPHEUS_CACHE_DIR}" ]]; then
    mkdir -p ${MORPHEUS_CACHE_DIR}
 fi
 
+if [[ "${MORPHEUS_SUPPORT_DOCA}" == "ON" ]]; then
```

Comment:
Suggestion: `docker/optional_deps/doca.sh:22` uses `@(TRUE|ON)`. Let's change one or the other so they match. Personally I'd prefer we stick to just one value (`ON`), like you have here already.
```suggestion
if [[ ${MORPHEUS_SUPPORT_DOCA} == @(TRUE|ON) ]]; then
```